### ISSUE TITLE
Timezone support

### ISSUE THREAD

**Issue by matsukaz** [matsukaz/hubot-schedule#4]
Currently, hubot-schedule does support timezone in datetime pattern.
Followings are the result which I run in my local machine that the timezone is set to JST.

```
Hubot> hubot schedule add "2015-10-16 10:00" Let's release this script!
1632: Schedule created
Hubot> hubot schedule add "2015-10-16 10:00 +00:00" Let's release this script! (with timezone)
1756: Schedule created
Hubot> hubot schedule list
1632: [ Fri Oct 16 2015 10:00:00 GMT+0900 (JST) ] #Shell Let's release this script!
1756: [ Fri Oct 16 2015 19:00:00 GMT+0900 (JST) ] #Shell Let's release this script! (with timezone)
```

But the cron pattern doesn't support setting timezone yet.
I'm thinking about supporting timezone settings in 2 ways.

1) setting timezone globally

```
env.HUBOT_SCHEDULE_TIMEZONE = JST
```

This will change the timezone from server time to specified timezone.

2) setting timezone each time

Like datetime pattern, setting timezone each time in cron pattern.

```
Hubot> hubot schedule add "0 10 * * 1-5"  hubot image me coffee
5321: Schedule created
Hubot> hubot schedule add "0 10 * * 1-5" "+00:00"  hubot image me coffee (with timezone)
5548: Schedule created
Hubot> hubot schedule list
5321: [ 0 10 * * 1-5 ] #Shell hubot image me coffee
5548: [ 0 10 * * 1-5 ] +00:00 #Shell hubot image me coffee (with timezone)
```

I'm quite busy now, so it may take a while to handle this issue.

**Comment by matsukaz** [comment]
I'm trying to change dependent module from node-schedule to node-cron, which supports timezone.

However, node-cron's timezone must be in format like 'Asia/Tokyo', not like '+09:00'.
I've sent a pr to use the later format, so if this pr is merged, I'll merge into hubot-schedule and suppoprt timezone.
https://github.com/ncb000gt/node-cron/pull/184

**Comment by mrala** [comment]
@matsukaz any plans on resolving the conflicts in your pr to node-cron?

**Comment by matsukaz** [comment]
@mrala sorry for leaving this issue for such a long time X(
I'll check the pr to resolve conflicts in next few days.

**Comment by joyarzun** [comment]
@matsukaz It would be good to change to node-cron although we should put the timezone format like 'Asia/Tokyo' while the PR 184 is merged. It's better that nothing.  What you think?

**Comment by matsukaz** [comment]
@joyarzun Yeah, I think so too.
I'll try to change the dependency to node-cron while sending request to pr again.

**Comment by hagel86** [comment]
Any update on this? Would be great to have!

**Comment by matsukaz** [comment]
Sorry for leaving this issue for such a long time.
I'm currently working on this, so please wait for a while.

**Comment by matsukaz** [comment]
@hagel86 @joyarzun @mrala 
I've just released new version `v0.7.0` that supports UTC Offset for cron-style pattern!
Sorry for leaving this issue for such a long time.
